Providers/PocketFeed.swift

```diff
@@ -0,0 +1,134 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Alamofire
+import Shared
+import Deferred
+import Storage
+
+private let PocketEnvAPIKey = "PocketEnvironmentAPIKey"
+private let PocketGlobalFeed = "https://getpocket.com/v3/firefox/global-recs"
+private let MaxCacheAge = OneMinuteInMilliseconds * 60 // 1 hour in seconds
```

Comment:
maybe type it as MaxCacheAge: TimeInterval so that seconds is implicit in the type